tests/components/fujitsu_hvac/snapshots/test_coordinator.ambr

```diff
@@ -0,0 +1,141 @@
+# serializer version: 1
+# name: test_coordinator_initial_data[climate.testserial123-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'fan_modes': list([
+      ]),
+      'hvac_modes': list([
+      ]),
+      'max_temp': 1.0,
+      'min_temp': 1.0,
+      'swing_modes': list([
+      ]),
+      'target_temp_step': 0.5,
+    }),
```

Comment:
It would be nice to have a full device in the fixtures so this is also reflecting real life so the test works as best as possible